src/common/interfaces.ts

```diff
@@ -70,7 +70,8 @@ export interface Tree {
   hausnr?: string | null;
   zusatz?: string | null;
   pflanzjahr?: string | null;
-  standalter?: string | null;
+  standalter?: string | null; // this may be redundant now, but I'm not entirely sure
```

Comment:
I searched for this property everywhere in the codebase and I think we can remove it altogether. Good catch! 🪲🔫

Reply:
Hmm, I've taken another look now and the type is apparently needed in `src/components/Sidebar/SidebarTree/TreeInfos.tsx`. Removing it strangely doesn't throw errors in dev mode, but I would be hesitant to remove it already...